content/applications/sales/subscriptions/automatic_alerts.rst

```diff
@@ -1,83 +1,159 @@
+applies.
 
-.. image:: automatic_alerts/create-a-new-automatic-alert.png
+To view or modify any existing automation rule, simply click the desired rule from this page.
+
+.. note::
+   When modifying an existing automation rule, Odoo "grays-out" the :guilabel:`Action` section of
+   the form, and provides the following warning: *Action data can not be updated to avoid unexpected
+   behaviors. Create a new action instead.*
+
+To create a new automation rule, click :guilabel:`New`.
+
+.. image:: automatic_alerts/automation-rules-page.png
   :align: center
-  :alt: New automatic alert in Odoo Subscriptions
-
-1. On the *Apply on* section, first give the alert a name. Then, you can choose to apply this alert
-   on a subscription template, on a specific customer, or even on a specific product. If you want
-   to add more specifications, you can also specify the value of your MRR, the change rate of your
-   MRR over a certain period of time, the value of the satisfaction rate, and even the stage to
-   which you want to apply this alert.
-
-   .. note::
-      In this example, the alert is applied to a specific product, and the stage goes from
-      *Undefined* to *In Progress*.
-
-2. For the *Action* section, specify the *Action* and the *Trigger on*. If the *Trigger on* is set
-   to *Modification*, the action is triggered every time there is a change or anything added to the
-   subscription, and all the conditions on the *Apply on* section are met. Now, if the *Trigger on*
-   is set to *Timed condition*, it means that the action is triggered based on the type of
-   *Trigger date*. After that, you can choose your *Action*. You have the choice between
-   *Create next activity*, *Set a tag on the subscription*, *Set a stage on the subscription*,
-   *Mark as To Renew*, *Send an email to the customer* and *Send an SMS Text Message to the
-   customer*.
-
-   .. note::
-      In the example above, the *Trigger on* is set to *Timed condition*, therefore, a
-      *Trigger date* and *Delay after trigger* need to be specified. And because the *Send an email
-      to the customer* action was adopted, an *Email template* can be chosen.
-
-As a result, this alert will send a rating survey after one month, to the customers who have
-purchased that specific product. The survey will appear in the chatter of your respective
-subscription.
-
-.. image:: automatic_alerts/rating-satisfaction-survey.png
+  :alt: The Automation Rules page in the Odoo Subscriptions application.
+
+Clicking :guilabel:`New` reveals a blank :guilabel:`Automation Rules` form with numerous fields to
+configure.
+
+.. image:: automatic_alerts/automation-rules-form.png
   :align: center
-  :alt: Satisfaction survey in Odoo Subscriptions
+  :alt: A sample Automation Rules form in the Odoo Subscriptions application.
+
+Automation rule form fields
+---------------------------
+
+- :guilabel:`Action Name`: title of the automated action rule.
+
+Apply On section
+~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Apply on
~~~~~~~~
```
possibility for brevity and sentence case formatting. Feel free to reject this, as I know you have it written to match the section in the Automation Rule form (: